context.go

```diff
@@ -124,6 +124,13 @@ func (c *Ctx) LoadProject(path string) (*Project, error) {
 //
 // The second returned string indicates which GOPATH value was used.
 func (c *Ctx) SplitAbsoluteProjectRoot(path string) (string, error) {
+	// the path may lie within a symlinked directory, resolve those first
+	path, err := filepath.EvalSymlinks(path)
```

Comment:
Instead of calling this within `SplitAbsoluteProjectRoot()`, let's do the symlink evaluation up in `LoadProject()`. That keeps this a bit cleaner, by keeping this function solely dealing with symbolic strings, and `LoadProject()` doing the actual filesystem work.